### PR TITLE
Massively buffs timestop effects

### PR DESCRIPTION
**Pull request by ChangelingRain**
> You can no longer click on the timestop effect itself.

That means you can heal yourself, hit people, etc, without hitting the timestop effect and looking like an absolute dork.

### CODE REVIEW

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -422,6 +422,7 @@
 	pixel_x = -64
 	pixel_y = -64
 	unacidable = 1
+	mouse_opacity = 0
```
Comment (Anonus) [inline code/modules/research/xenobiology/xenobiology.dm]: I AM LITERALLY ONE-LINE FIXING, HERE, AND THIS ONE LINE IS A BUFF THE SIZE OF NEVADA

**Comment by duncathan** [comment]
>Anonus:switchblades

Also you should probably changelog this.

**Comment by Anonus** [comment]
Well, it is a buff to switchblades in the context of timestop :^)

**Comment by Incoming5643** [comment]
did it really need the buff?

**Comment by KorPhaeron** [comment]
>did it really need the buff?

**Comment by Zilenan91** [comment]
It's a 20 brute weapon that's like 10 influence, I don't think it needed a buff.

**Comment by Anonus** [comment]
I never see anyone take it, so maybe I dunno. This is a buff to scientists if they get septa slimes, though they don't often.

**Comment by Zilenan91** [comment]
Pistols, uzis, and uniforms are usually better than a melee weapon.

**Comment by Jalleo** [comment]
What zilenan91 said

**Comment by Robustin** [comment]
You idiots actually managed to derail into a switchblade discussion.

Anon is still talking about Timestop, which is all that's being changed here. 

While it was a quality of life issue, IMO keeping timestop to have some sort of unclickability overlay was/is a fair balancing point. Plenty of people take Timestop and it's quickly seeing acceptance among combat wizards as another spell for murdering people with 0 counterplay.

With magic missile, jaunt, and other CC spells/avoidance spells (i.e. lightning) you start reaching game-breaking levels of invulnerability. Timestop is in that really imba territory of "extreme defense AND offense potential". You can use it to avoid what might be a deadly crossfire of lasers, or just pop it next to ol' Captain and weld him to death while laughing at all the helpless bystanders.

In short, this didn't need a buff, and you're going to have to come up with a better way to balance this in return for making it even easier to use.

**Comment by Anonus** [comment]
It 's not a very effective balance point, because if you catch someone on the edge or they run out of it, it still stuns them and you can still attack them, which makes it less of a balance point and more "this is really fucking clunky".
In addition, if you altclicked the tile you're on(or the tile you want to fuck with) before casting it you can still heal yourself/do stuff to that tile, which is... pretty bad.

My preference would be to make it last much less time so that, even if you've got it at max, you can't use it to heal while the crew watches in anger because you'll get hit by lasers the second it ends and it'll be ending much more often.

**Comment by KorPhaeron** [comment]
But if it lasts less time you can't stack lasers in it. 

Which is like the entire point of the thing. Otherwise its just a stun.

As to people caught on the edge the effect needs to be bigger visually

**Comment by Robustin** [comment]
Yea no doubt it was clunky before, but observing what wizards can do to people caught in the edge... not really excited that the one-sided beatdown will be available across the whole area of affect now. 

With a decent melee weapon timestop will essentially be ei nath + MM, two of wizard's most powerful staples, merged into a single spell, minus the gibbing.

**Comment by Zilenan91** [comment]
The wizard can also combo this with mutate and fire barrages of lasers down hallways all at once.

**Comment by Anonus** [comment]
The wizard can already do that, this doesn't change the ability to do that.

**Comment by Dorsisdwarf** [comment]
Do flashbangs thrown into the timestop pause their detonation?

If so, just throw a bunch in, and the wiz either leaves his timefort or gets cucked

**Comment by Razharas** [comment]
Makes sense thb fams
